common/djangoapps/student/views.py

```diff
@@ -1612,6 +1612,7 @@ def _do_create_account(form, custom_form=None):
     ]
     profile = UserProfile(
         user=user,
+        usa_region=form.data['usa_states'] if 'usa_states' in form.data.keys() else None,
```

Comment:
1. why do you use `form.data` instead of `form.cleaned_data`?
2. I think this row will be reduntant if you use `form.cleaned_data`